int32 target
float32[] timelinep
float32[] intentp
float32[] timeliner
float32[] intentr
string color